core/roslib/src/roslib/packages.py

```diff
@@ -384,20 +384,24 @@ def list_pkgs_by_path(path, packages=None, cache=None, env=None):
             
     return packages
 
-def find_node(pkg, node_type, rospack=None):
+def find_node(pkg, node_type, rospack=None, source_path_to_packages=None):
     """
     Warning: unstable API due to catkin.
 
     Locate the executable that implements the node
     
     :param node_type: type of node, ``str``
+    :param source_path_to_packages: the dictionary is populated with mappings
+        from source paths to packages.
```

Comment:
Please use the same documentation as in https://github.com/ros/catkin/blob/3b8a3b9607adff98ed32cfd3b6d6ea4aed647424/python/catkin/find_in_workspaces.py#L106

Same below.